src/http_module.cpp

```diff
@@ -548,7 +524,14 @@ ngx_int_t initModule(ngx_conf_t* cf)
     *h = onRequestEnd;
 
     gpr_set_log_function(grpcLogHandler);
+#if USE_ABSL_LOGGING
+    absl::InitializeLog();
```

Comment:
According to its doc it shouldn't be called twice, but here we might all it twice on config reload.